```
StackOverflowError()
```

Der Funktionsaufruf wuchs über die Größe des Aufrufstapels hinaus. Dies geschieht normalerweise, wenn ein Aufruf unendlich rekursiv ist.
